### PR TITLE
Adds CLI for Microsoft 365 v10.1 release blog

### PR DESCRIPTION
**Pull request by milanholemans**
## Category

- [ ] Content fix
- [x] New article

## Related issues

None

## Contents of the Pull Request

This is a new blog post about the new CLI for the Microsoft 365 release.

### CODE REVIEW

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+- [Martin Lingstuyl](https://github.com/martinlingstuyl)
+- [Martin Machacek](https://github.com/MartinM85)
+- [Mathijs Verbeeck](https://github.com/MathijsVerbeeck)
+- [Michał Kornet](https://github.com/mkm17)
+- [Milan Holemans](https://github.com/milanholemans)
+- [Nanddeep Nachan](https://github.com/nanddeepn)
+- [Reshmee Auckloo](https://github.com/reshmee011)
+- [Saurabh Tripathi](https://github.com/Saurabh7019)
+- [Shantha Kumar T](https://github.com/ktskumar)
+- [Waldek Mastykarz](https://github.com/waldekmastykarz)
```
Comment (Adam-it) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md:173]: not that I want to be at the top of the list but it does say alphabetical order 😅

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+docker run --rm -it m365pnp/cli-microsoft365:latest
+```
+
+## Need More Information?
+
+For additional guidance on getting started or to explore detailed information about commands, architecture, or the project itself, visit [aka.ms/cli-m365](https://aka.ms/cli-m365).
+
+## Stay Connected!
+
+We value your feedback and are eager to hear from you. If you have any suggestions for improvement or want to engage with our community, you can reach out to us on [GitHub](https://github.com/pnp/cli-microsoft365/issues), [Discord](https://aka.ms/cli-m365/discord), or [X](https://x.com/climicrosoft365). Don't hesitate to connect with us. Your input plays a vital role in shaping the future of CLI for Microsoft 365.
```
Comment (Adam-it) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md:201]: maybe we could also mention our bluesky account? https://bsky.app/profile/climicrosoft365.bsky.social

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+- [entra m365group get](https://pnp.github.io/cli-microsoft365/cmd/entra/m365group/m365group-get)
+- [graph subscription add](https://pnp.github.io/cli-microsoft365/cmd/graph/subscription/subscription-add)
+- [spo list contenttype list](https://pnp.github.io/cli-microsoft365/cmd/spo/list/list-contenttype-list)
+- [spo contenttype list](https://pnp.github.io/cli-microsoft365/cmd/spo/contenttype/contenttype-list)
+- [spo contenttype get](https://pnp.github.io/cli-microsoft365/cmd/spo/contenttype/contenttype-get)
+
+If you are eager to go over all of the details and improvements added to this release, do not hesitate to check out the [release notes](https://pnp.github.io/cli-microsoft365/about/release-notes#v1010) to learn more.
+
+## Upcoming Changes
```
Comment (Adam-it) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md:151]: maybe we could add a small remark that CLI just turned 6 years old 🤩😍

**Review by Adam-it — APPROVED** [review]
Awesome work. 👏👏👏👏
I added some minor remarks we may ignore 😅

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md (hunk 3 above)
Comment (milanholemans) [reply content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md:151]: It's 7 years, no?

Comment (Adam-it) [reply content/cli-for-microsoft-365/cli-for-microsoft-365-v10-1/index.md:151]: aaa yes,... wow time passes by so fast 👴

**Pull request merged**